fastlane/Fastfile

```
@@ -0,0 +1,78 @@
+
+  suffix = suffixes[flavor] || ""
+  "#{base_package}#{suffix}"
+end
+
+
+default_platform(:android)
+platform :android do
+
+  desc "Get App Version" 
+  lane :get_app_version do 
+    app_version = android_get_version_name
```

rubocop (1.69.1)

[warning] Useless assignment to variable - `app_version`.

(Lint/UselessAssignment)